authors/apps/articles/test/test_create_article.py

```diff
@@ -0,0 +1,181 @@
+import json
```

Comment:
Directory name should be `tests` since you'll have more tests. The tests in this file are not only for creating articles, yet your file_name and class name says it's only about creating articles.